docs/coreConcepts/Wallets/overview.md

```diff
@@ -0,0 +1,85 @@
+
+The two-thirds threshold design provides a level of censorship resistance and fault tolerance that “typical” 2-of-2 MPC designs (e.g. Fireblocks) do not. In addition to any 2-of-2 provider being able to deny the user access to their funds or censor transactions, most of these systems also require the end user to custody a key share. This means the goal of a seamless, “web2” style onboarding UX is not possible (onboarding without seed phrases or private key management), instead delivering the UX of self-custody with additional steps.
+
+In Lit’s model, the entire key lives in the network, and *any* arbitrary rules for [authentication](https://developer.litprotocol.com/coreConcepts/LitActionsAndPKPs/actions/authHelpers) can be assigned to that key pair through the use of [Lit Actions](https://developer.litprotocol.com/coreConcepts/LitActionsAndPKPs/actions/litActions). Authentication refers to the *****method***** that “owns” or controls the underlying key, denoting who has the power to combine the shares. This means that the provider or end user have full control over designing how these interactions should be managed. For example, allowing a user to create a wallet with nothing but their Gmail account, or requiring multi-factor authentication (MFA) when attempting to spend more than X amount of assets, as well as enabling flexible social recovery. Today, PKPs support the following auth methods:
+
+- WebAuthn from [FIDO Alliance](https://fidoalliance.org/fido2-2/fido2-web-authentication-webauthn/) (AKA Apple Passkey, [demo](http://getlit.dev/demo))
+- Web3 ownership (holding the “controller” [PKP NFT](https://explorer.litprotocol.com/mint-pkp))
+- oAuth (Google, Discord)
+- Self-custody key (such as a Ledger hardware wallet)
+
+The current methods are a work in progress:
+
+- Email
+- SMS
+
+You can read more about how authentication works with PKPs in [this section](https://developer.litprotocol.com/coreConcepts/LitActionsAndPKPs/actions/authHelpers) of the docs.
+
+The end goal of this system is to facilitate a simple onboarding experience that helps bridge the next billion users into Web3.
+
+![authOverview](/img/authOverview.png)
+
+## Supported Chains
+
+Today, PKP wallets are inherently compatible with any blockchain or cryptographic system that utilizes [ECDSA](https://blog.cloudflare.com/ecdsa-the-digital-signature-algorithm-of-a-better-internet/) for digital signatures. This includes Ethereum and most EVM chains, Bitcoin, Cosmos-based chains, and storage networks like IPFS. This means that PKPs can be used to read and write data across these networks, bringing interoperability to previously disconnected ecosystems. You can view the complete list of supported chains [here](https://developer.litprotocol.com/Support/supportedChains#programmable-key-pairs).
+
+## Using oAuth for "Seed-Phraseless" Onboarding
```

Comment:
new page starting at this heading, the new page should be titled something like `initial implementation`